src/app/generic-listing-page/generic-listing-page-routing.module.ts

```diff
@@ -13,6 +13,11 @@ const routes: Routes = [
     path: 'saved',
     component: GenericListingPageComponent,
     data: listingConfig.saved
+  },
+  {
+    path: 'recommended',
```

Comment:
change it to recommendations

Reply:
done

Reply:
is the path changed?

Reply:
yea

Reply:
here the path declared is "recommended" but when navigating from home-page to this page this path is given "/list/recommendations", will this navigation work. Please check